Attendee.js

```diff
@@ -0,0 +1,43 @@
+// The Attendee class can be utililized to keep certain information of the attendee, 
+// a class for the user needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
+class Attendee {
+    constructor() {
+        this.sessionId;
+        this.screenName;
+        this.TimeLastPolled;
+    }
+        
+    setSessionId(id) { 
+        this.sessionId = id;
+    }
+
+    setScreenName(name) {
```

Comment:
These setters should be private. See the style guide for how to denote private functions. We dont expect or want someone to mutate these objects after creation, right?